openadapt/models.py

```diff
@@ -519,6 +577,22 @@ def __init__(
         self.initialize_instance_attributes()
         self._image = image
 
+    def scrub(self, scrubber: ScrubbingProvider) -> None:
+        """Scrub the screenshot."""
+        logger.info(f" -> Scrubbing screenshot id: {self.id}")
```

Comment:
If you check `record.py` there is a utility that automatically logs entering and exiting like this. What do you think about re-using it here?